test/common.js

```diff
@@ -12,13 +12,15 @@ var rimraf = require('rimraf')
 var fs = require('fs')
 var path = require('path')
 var delayed = require('delayed').delayed
-var levelup = require('../lib/levelup.js')
+
+var levelup = require('../lib/levelup.js').default
+var leveluperrors = require('../lib/levelup.js').errors
 var errors = require('level-errors')
 var dbidx = 0
-var leveldown = require('leveldown')
-var encDown = require('encoding-down')
+var leveldown = require('leveldown').default
+var encDown = require('encoding-down').default
 
-assert(levelup.errors === errors)
+assert(leveluperrors === errors)
```

Comment:
Is this a requirement for TS? I like the previous better

Reply:
unfortunately, without moving the test to ts (so you can trick the compiler) I'm not sure. the alternative would be to have specific typescript-based tests for common things - that way the typescript stuff would not interfere at all with the js tests. I don't mind doing that with something like mocha.

Reply:
I'll have another stab; might be a way to add the errors onto the default too.